src/main/java/org/mule/extension/http/internal/listener/profiling/tracing/HttpListenerCurrentSpanCustomizer.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.extension.http.internal.listener.profiling.tracing;
+
+import static java.lang.String.valueOf;
+
+import static com.google.common.net.HttpHeaders.USER_AGENT;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import org.mule.extension.http.api.HttpRequestAttributes;
+import org.mule.extension.http.internal.request.profiling.tracing.HttpCurrentSpanCustomizer;
+import org.mule.sdk.api.runtime.source.DistributedTraceContextManager;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.slf4j.Logger;
+
+/**
+ * A customizer for the current span for HTTP Requests
+ *
+ * @since 1.8.0
+ */
+public class HttpListenerCurrentSpanCustomizer extends HttpCurrentSpanCustomizer {
+
+  private static final Logger LOGGER = getLogger(HttpListenerCurrentSpanCustomizer.class);
+
+  public static final String HTTP_TARGET = "http.target";
```

Comment:
Isn't the http.scheme attribute missing? Could it be easily added?

Reply:
It was missing. Thanks. Pushed.